### PR TITLE
release: codeready-toolchain-operator 0.1.0

### PR DESCRIPTION
**Pull request by MatousJobanek**
see https://github.com/operator-framework/community-operators/pull/996

Signed-off-by: Matous Jobanek <mjobanek@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -2,6 +2,29 @@ apiVersion: operators.coreos.com/v1alpha1
...
+      [
+        {
+          "apiVersion": "toolchain.openshift.dev/v1alpha1",
+          "kind": "CheInstallation",
+          "metadata": {
+            "name": "toolchain-cheinstallation"
+          },
+          "spec": {
+            "cheOperatorSpec": {
+              "namespace": "toolchain-che"
```
Comment (xcoulon) [inline deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml]: I believe it should be `toolchain-workspaces`

```suggestion
              "namespace": "toolchain-workspaces"
```

File: examples/toolchain.openshift.dev_cheinstallations_cr.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: toolchain.openshift.dev/v1alpha1
+kind: CheInstallation
+metadata:
+  name: toolchain-cheinstallation
+spec:
+  cheOperatorSpec:
+    namespace: toolchain-che
```
Comment (xcoulon) [inline examples/toolchain.openshift.dev_cheinstallations_cr.yaml]: same here:

```suggestion
    namespace: toolchain-workspaces
```

File: hack/deploy_csv.yaml [hunk 3]
```diff
@@ -209,6 +209,29 @@ data:
...
+            [
+              {
+                "apiVersion": "toolchain.openshift.dev/v1alpha1",
+                "kind": "CheInstallation",
+                "metadata": {
+                  "name": "toolchain-cheinstallation"
+                },
+                "spec": {
+                  "cheOperatorSpec": {
+                    "namespace": "toolchain-che"
```
Comment (xcoulon) [inline hack/deploy_csv.yaml]: same here:
```suggestion
                    "namespace": "toolchain-workspaces"
```

File: manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+      [
+        {
+          "apiVersion": "toolchain.openshift.dev/v1alpha1",
+          "kind": "CheInstallation",
+          "metadata": {
+            "name": "toolchain-cheinstallation"
+          },
+          "spec": {
+            "cheOperatorSpec": {
+              "namespace": "toolchain-che"
```
Comment (xcoulon) [inline manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml]:
```suggestion
              "namespace": "toolchain-workspaces"
```

File: manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: CheInstallation defines how CodeReady Workspaces (Che) operator
+        should be installed
+      displayName: CodeReady Toolchain Che Installation
+      kind: CheInstallation
+      name: cheinstallations.toolchain.openshift.dev
+      specDescriptors:
+      - description: The namespace where the Che operator will be installed
```
Comment (xcoulon) [inline manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml]:
```suggestion
      - description: The namespace where the CodeReady Workspaces operator will be installed
```

File: manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+        displayName: Namespace
+        path: namespace
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      statusDescriptors:
+      - description: Last known condition of the che operator installation
```
Comment (xcoulon) [inline manifests/0.1.0/codeready-toolchain-operator.v0.1.0.clusterserviceversion.yaml]:
```suggestion
      - description: Last known condition of the CodeReady Workspaces operator installation
```

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+                    description: Last time the condition transit from one status to
+                      another.
+                    format: date-time
+                    type: string
+                  message:
+                    description: Human readable message indicating details about last
+                      transition.
+                    type: string
+                  reason:
+                    description: (brief) reason for the condition's last transition.
```
Comment (tinakurian) [inline manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]:
```suggestion
                    description: Brief reason for the condition's last transition.
```

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+        metadata:
+          type: object
+        spec:
+          description: TektonInstallationSpec defines the desired state of TektonInstallation
+          type: object
+        status:
+          description: TektonInstallationStatus defines the observed state of TektonInstallation
+          properties:
+            conditions:
+              description: 'Conditions is an array of current TektonInstallation conditions
```
Comment (tinakurian) [inline manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]:
```suggestion
              description: 'Conditions is an array of current TektonInstallation conditions.
```

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+                Supported condition types: TektonReady'
+              items:
+                properties:
+                  lastTransitionTime:
+                    description: Last time the condition transit from one status to
```
Comment (tinakurian) [inline manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]:
```suggestion
                    description: Last time the condition transitioned from one status to
```

File: manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+              - namespace
+              type: object
+          required:
+          - cheOperatorSpec
+          type: object
+        status:
+          description: CheInstallationStatus defines the observed state of CheInstallation
+          properties:
+            conditions:
+              description: 'Conditions is an array of current CheInstallation conditions
```
Comment (tinakurian) [inline manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]:
```suggestion
              description: 'Conditions is an array of current CheInstallation conditions.
```

File: manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
```
[the remark this answers is not in the document]
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: This is generated by operator-sdk - we cannot change it

File: manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+        status:
+          description: CheInstallationStatus defines the observed state of CheInstallation
+          properties:
+            conditions:
+              description: 'Conditions is an array of current CheInstallation conditions
+                Supported condition types: CheReady'
+              items:
+                properties:
+                  lastTransitionTime:
+                    description: Last time the condition transit from one status to
```
[the remark this answers is not in the document]
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: This is taken/generated from api repo - please change it there first

File: manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+                      another.
+                    format: date-time
+                    type: string
+                  message:
+                    description: Human readable message indicating details about last
+                      transition.
+                    type: string
+                  reason:
+                    description: (brief) reason for the condition's last transition.
```
[the remark this answers is not in the document]
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: same as the previous one - taken from api repo

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
```
[the remark this answers is not in the document]
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]: generated by operator-sdk

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml (hunk 9 above)
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]: taken from api repo

File: manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml (hunk 7 above)
Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_tektoninstallations.yaml]: taken from api repo

**Pull request merged**